### ISSUE TITLE
Graphical issue: Everything at Figshare is not necessarily a PDF

### ISSUE THREAD

**Issue by rossmounce** [dissemin/dissemin#345]
I have screenshotted 3 items here from my profile. They are all metadata ORCID records of research outputs at Figshare. The issue is the graphic on the left hand side says "PDF" with an OA logo. The OA logo is correct, but none of these three items are PDF-format. So keep the OA logo but don't use "PDF" logo unless the deposit file actually is a PDF.

![2017-02-01-123813_1366x744_scrot](https://cloud.githubusercontent.com/assets/824120/22507057/8b3c5b5e-e87b-11e6-86a5-4c08d272c1ac.png)

**Comment by wetneb** [comment]
You're right, we're using the word "PDF" for "full text", which is quite bad ^^
